### ISSUE TITLE
documentType.csv: financeArrangements and guaranteeReports have no Description

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/public-private-partnerships#126]

**Comment by jpmckinney** [comment]
@duncandewhurst Should we fill in these missing descriptions while we're updating PPPs?

**Comment by duncandewhurst** [comment]
These codes (and their titles) were based on a couple of bullet points in the [PPP disclosure framework](http://pubdocs.worldbank.org/en/773541448296707678/Disclosure-in-PPPs-Framework.pdf) (see page 59) for which no further definition or description were provided.

For the terms used in the title of the `financeArrangements` code ("Security and step in arrangements") the PPP Knowledge Lab [glossary](https://pppknowledgelab.org/glossary) gives the following definitions:

> **Step-in Rights**
> The right of a third party to "step in" to the place of one contractual party where that party fails in its obligations under the contract and the other party to the contract has the right to terminate the contract.

> **Security**
> A legal right of access to value through mortgages, contracts, cash accounts, guarantees, insurances, pledges or cash flow, including licences, concessions and other assets. Lenders can act on security rights in the event of default by the borrower. A negotiable certificate evidencing a debt or equity obligation/shareholding.

The above definition of "Step-in rights" is a broader definition than the use of the term in Infrastructure Australia's [National Public Private Partnership Policy and Guidelines](https://ausinf.affinitext.com/viewer/book?id=5322&toc_uid=1249502&context=ausinf) (free registration required), which seems clearer to me:

> Government may assume all or some of the service delivery obligations of the private party for a period of time

Seeking input on whether the PPP profile should use a description based on the broader ("third party step-in") or narrower ("government/private party step-in") definition.

**Comment by duncandewhurst** [comment]
For `guaranteeReports` code (title: "Fiscal commitment and contingent liabilities disclosure reports") I propose adding the following description, based on the definitions quoted below:

> Reports detailing the fiscal commitments of government to the PPP, these commitments can include known payments which must be made if the PPP proceeds and payments whose occurrence, timing and magnitude depend on uncertain future events outside the control of government.

From [the PPP Knowledge Lab glossary](https://pppknowledgelab.org/glossary):

> **Contingent Liability**
> A liability which is uncertain as to its crystallization, e.g. a guarantee or a contingent debt, either in amount and/or timing.

From [The APMG Public-Private Partnerships Certification Program](https://ppp-certification.com/ppp-certification-guide/181-types-fiscal-commitment-ppps):

> Fiscal commitments to PPPs can be payments for services, capital contributions, or subsidies to reduce costs for users, or a means to share risk. The wide range of fiscal commitments can usefully be divided into the following categories.
> 
>* Direct liabilities: known payments that must be made if the PPP proceeds (although there may be some uncertainty regarding the value).
>* Contingent liabilities: payment commitments whose occurrence, timing and magnitude depend on some uncertain future event, outside the control of government.
>* Liabilities of government owned off-takers

**Comment by jpmckinney** [comment]
# financeArrangements

Title is good. Let's use the broader definition. It's anticipated that some jurisdictions will have narrower definitions, like Australia.

# guaranteeeReports

Title: Fiscal commitment**s** and contingent liabilities disclosure reports

The APMG page is useful, so rather than choose which information to repeat and risk being incomplete or misleading, why not:

> The APMG PPP Certification Program describes [types of fiscal commitment to PPPs](https://ppp-certification.com/ppp-certification-guide/181-types-fiscal-commitment-ppps).

**Comment by duncandewhurst** [comment]
# financeArrangements

I'm uncomfortable with the broader definition, since my reading of it precludes the narrower definition, i.e. "third party" step-in would mean a party which was neither the government nor the contractor assuming the responsibilities of a party to the contract, whereas my understanding of step-in rights in the PPP context was that it related to government (not a third party) stepping in when the contractor failed to fulfill it's commitments.

I'm not a contract law or PPP expert though, so I may be assigning the wrong meaning to third party in this context.

# guaranteeReports

Linking out to the primary definition of a field or code isn't an approach we've taken generally, we could do that if we're happy that:

* The URL could become broken (do we check for that in a test when building the docs?)
* Applications which present the schema descriptions in an interface would need to permit users to click and follow the hyperlink to understand what the code should be used for.

**Comment by jpmckinney** [comment]
### financeArrangements

I briefly reviewed step-in rights. Making the party the government is too narrow, become it is often a lender. The part about "the other party to the contract has the right to terminate the contract" seems to be unnecessary and not always true. Outside PPPs, step-in rights allow a party to prosecute patent rights if another party with primary responsibility is disinterested.

The literature describes 'step-in' as having no standard definition, so I recommend keeping it loose like: "Any rights of a party to "step in" to the shoes of another party, for example, the right of a public authority to take over temporarily the operation of the project in case of failure to perform by the private party."

http://www.uncitral.org/pdf/english/yearbooks/yb-2000-e/yb_2000_e.pdf has some good paragraphs explaining the rights.

### guaranteeReports

There are more types of fiscal commitments than in the proposed definition, and it doesn't make sense to have a laundry list definition. We can instead not define fiscal commitment or contingent liability (which is a type of fiscal commitment) in the field's description – though perhaps a good idea to define it elsewhere. So leaving it as:

> Reports disclosing the fiscal commitments and contingent liabilities of the public authority to the PPP.

**Comment by nadinefernz** [comment]
Specifically for PPP projects, I find the World Bank definition quite useful and clear:

Step-in rights are rights given to lenders in project financed arrangements to "step in" to the project company's position in the contract to take control of the infrastructure project where the project company is not performing. There may be prohibitions in the law on lenders having step-in rights.

**Comment by jpmckinney** [comment]
^ from https://ppp.worldbank.org/public-private-partnership/legislation-regulation/framework-assessment/legal-environment/lender-issues-step-in-rights

However, that definition is specific to lenders, but there are step-in rights I've seen in PPPs in which the public authority has step-in rights. We'd probably remove the sentence about prohibitions in the law as it is not necessary to the definition.

@nadinefernz What do you think of this?

> Any rights granted to a party to "step in" to the shoes of another party, for example, a right granted to the public authority to take over temporarily the operation of the project in case of failure to perform by the private party.

**Comment by duncandewhurst** [comment]
Okay great, thanks both. I think the discussion so far has been useful in clarifying what descriptions in OCDS should and shouldn't aim to do.

For `financeArrangements`, I've taken what I think are the two key points from the PPP Knowledge Lab definition of security, omitting the long list of different mechanisms, and combined that with the step-in rights definition from @jpmckinney to give:
 
> Details of any rights granted to a lender to access value in the event of default by a borrower and any rights granted to a party to "step in" to the shoes of another party, for example, a right granted to a lender or public authority to take over temporarily the operation of a project in case of failure to perform by a private party.

For `guaranteeReports`, I think we should consider the needs non-PPP expert data users, for whom the description of the code might serve to help them understand what a document attached to an OCDS release describes, or whether it is likely to contain the type of information they are looking for.

I don't think the proposed description, which just repeats the title of the code, meets that need, so I propose the following, which makes clear the description is giving examples rather than a laundry list.

> Reports detailing the fiscal commitments of the public authority to the PPP, for example known payments which must be made if the PPP proceeds and payments whose occurrence, timing and magnitude depend on uncertain future events outside the control of the public authority.

Basically I think it's useful to break down what contingent liabilities means into (relatively) plain English.
